test/tests/misc/exception/tests

```diff
@@ -89,16 +83,26 @@
     # The mesh is too small, and the number of elements on MPI rank 1 is 0 when we are uing 16 MPI processes. This happens with PETSc-3.8.3
     max_parallel = 10
 
-    requirement = 'The system shall support throwing an error during a Jacobian '
-                  'calculation, which will terminate the solve.'
-  [../]
+    requirement = 'The system shall support throwing an error during a Jacobian calculation, which will terminate the solve.'
+  []
 
-  [./skip_exception_check]
+  [skip_exception_check]
     type = 'Exodiff'
     input = '2d_diffusion_skip_exception.i'
     exodiff = '2d_diffusion_skip_exception_out.e'
     expect_out = 'MOOSE may fail to catch an exception when the "skip_exception_check" parameter is used. If you receive a terse MPI error during execution, remove this parameter and rerun your simulation'
     allow_warnings = true
-    requirement = "The system shall allow users to skip exception checks to avoid global communication."
-  [../]
+    requirement = "The system shall allow users to skip exception checks to avoid global "
+                  "communication."
+  []
+
+  [exception_transient]
+    type = RunApp
+    # type = 'CSVDiff'
+    input = 'exception_transient.i'
+
+    allow_warnings = true
+    allow_test_objects = true
+    requirement = "The system shall interrupt a solve immediately as soon as an exception is thrown."
```

Comment:
CSVDiff or runexception would do better

Reply:
D'oh, of course it should be CSVDiff, that's why I checked the gold file in :-D